docs/static/settings/monitoring-settings.asciidoc

```diff
@@ -86,23 +93,29 @@ Optional settings that provide the password to the keystore.
 Option to validate the server’s certificate. Defaults to `certificate`. To
 disable, set to `none`. Disabling this severely compromises security.
 
+`monitoring.elasticsearch.sniffing`::
+
+Finds all {es} cluster nodes and adds them to the hosts list.
+
+Defaults to `false`.
```

Comment:
Do you need the newline here? It renders slightly weirdly
<img width="727" alt="Screen Shot 2020-04-09 at 11 31 16 AM" src="https://user-images.githubusercontent.com/4061337/78912582-c2f70980-7a55-11ea-9141-ba866e4e1865.png">